addons/web_editor/static/src/scss/wysiwyg.scss

```diff
@@ -695,10 +695,31 @@ img.o_we_selected_image {
     user-select: none;
     box-shadow: $box-shadow;
     border-color: rgba(0,0,0,0.025);
+    background-color: $o-we-popover-bg;
 
-    .popover-arrow::before {
-        border-right-color: rgba(0, 0, 0, 0.05);
```

Comment:
No idea what the purpose of this was. It led to a visual bug.
![image](https://github.com/user-attachments/assets/b6195c34-26ef-425c-bb51-8667d4464857)